README.md

```diff
@@ -33,7 +33,7 @@ We’ve put together some commonly asked questions and answers about the Camunda
 ### Contributor resources & documentation
 * [Contributor guide](https://github.com/camunda-community-hub/community/blob/main/CONTRIBUTING.MD)
 * [Issue triage and labeling](https://github.com/camunda-community-hub/community/blob/main/issue-triage.md)
-* [Maintainer & reviewer expectations]([#maintainer-reviewer-expectations.md](https://github.com/camunda-community-hub/community/blob/main/maintainer-reviewer-expectations.md))
+* [Maintainer & reviewer expectations](https://github.com/camunda-community-hub/community/blob/main/maintainer-reviewer-expectations.md)
```

Comment:
Does `[Maintainer & reviewer expectations](./maintainer-reviewer-expectations.md)` work here?

Reply:
This one *should* work if I do that, but I was avoiding relative links to make sure I got it right. Is it better practice to use relative links?

Reply:
If the files are within the same repo, yes.

Reply:
Updated!